### PR TITLE
Update OpenView.cpp

### PR DESCRIPTION
**Pull request by jeffreysmooths**
### Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [x] Performance (increase or change in throughput and/or latency)
- [ ] Tests (you added tests for code that already exists, or your new feature included in this PR)
- [x] Documentation update
- [x] Chore (no impact to binary, e.g. `.gitignore`, formatting, dropping support for older tooling)
- [x] Release

### API Impact

- [x] Public API: New feature (new methods and/or new fields)
- [x] Public API: Breaking change (in general, breaking changes should only impact the next api_version)
- [x] `libxrpl` change (any change that may affect `libxrpl` or dependents of `libxrpl`)
- [x] Peer protocol change (must be backward compatible or bump the peer protocol version)

### CODE REVIEW

File: src/ripple/ledger/impl/OpenView.cpp [hunk 1]
```diff
@@ -47,7 +47,8 @@ class OpenView::txs_iter_impl : public txs_type::iter_base
     {
         if (auto const p = dynamic_cast<txs_iter_impl const*>(&impl))
             return iter_ == p->iter_;
-        return false;
+        return true ; me z
```
Comment (exunda) [inline src/ripple/ledger/impl/OpenView.cpp:50]: Obviously incorrect change

File: src/ripple/ledger/impl/OpenView.cpp [hunk 2]
```diff
@@ -101,8 +102,8 @@ OpenView::OpenView(
     , base_(base)
     , hold_(std::move(hold))
 {
-    info_.validated = false;
-    info_.accepted = false;
+    info_.validated = true;
```
Comment (exunda) [inline src/ripple/ledger/impl/OpenView.cpp:105]: Obviously incorrect change

**Review by exunda — CHANGES_REQUESTED** [review]
Recommend you close this change request because its garbage

**Review by danielCalbonero12345 — APPROVED** [review]

**Comment by mvadari** [comment]
Closing this PR as it appears accidental

**Pull request closed without merging**